automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/LeonardoSuite.scala

```diff
@@ -90,7 +90,7 @@ trait GPAllocUtils extends BillingFixtures with LeonardoTestUtils {
       _ <- IO(logger.info(s"Single test project $project claimed"))
       t <- testCode(project)
       _ <- unclaimProject(project)
-      _ <- IO(logger.info(s"releasing single-test project: ${project.value}"))
+      _ <- IO(logger.info(s"not releasing single-test project: ${project.value}"))
```

Comment:
Why the `not`? I think the project is released by this method, no?

Reply:
oops, i commented out the above line for testing, missed this